modules/UI/videolayout/RemoteVideo.js

```diff
@@ -132,7 +133,7 @@ RemoteVideo.prototype._generatePopupContent = function () {
     let popupmenuElement = document.createElement('ul');
     popupmenuElement.className = 'popupmenu';
     popupmenuElement.id = `remote_popupmenu_${this.id}`;
-    let menuItems = [];
+    let menuLinkItems = [];
```

Comment:
Why the rename? IMHO the previous name was ok, the rename adds unnecessary noise to this PR while not adding much.